buildtools/webpack.commons.js

```diff
@@ -134,6 +134,7 @@ const config = {
       'moment': 'moment/src/moment.js',
       'ol/ol.css': 'openlayers/css/ol.css',
       'ol': 'openlayers/src/ol',
+      'goog': 'olcs/goog/',
```

Comment:
Why this?

Reply:
You are right, this is a remaining from when I was on an old branch.

Reply:
ok :-)